### ISSUE TITLE
Install Issue 0.9.6

### ISSUE THREAD

**Issue by Pingblue** [rakshasa/rtorrent#808]
I'm running Ubuntu 18.04 and currently have libtorrent/rtorrent 0.9.7 installed but keep getting connection issues between rtorrent/rutorrent so wanted to downgrade but having issue installing 0.9.6

libcppunit-dev is installed.

Any help would be appreciated.

```
aclocal...
configure.ac:22: warning: macro 'AM_PATH_CPPUNIT' not found in library
autoheader...
libtoolize... using libtoolize
automake...
configure.ac:3: installing './compile'
configure.ac:20: installing './missing'
src/Makefile.am: installing './depcomp'
autoconf...
configure.ac:22: error: possibly undefined macro: AM_PATH_CPPUNIT
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.

```

**Comment by phxn** [comment]
Try with these dependencies:

`apt update && apt upgrade && apt -y install automake build-essential curl g++ gcc git intltool libcppunit-dev libcurl4-openssl-dev libncurses5-dev libsigc++-2.0-dev libssl-dev libtool libxml2-dev lsb-release xdg-utils zlib1g-dev
`

**Comment by Pingblue** [comment]
I get the same error after installing these dependencies.

**Comment by pyroscope** [comment]
https://bintray.com/pyroscope/rtorrent-ps/download_file?file_path=rtorrent-ps_0.9.6-PS-1.1-25-g1b53ac9%7Ejessie_amd64.deb

dpkg -i that

**Comment by pyroscope** [comment]
err, https://bintray.com/pyroscope/rtorrent-ps/download_file?file_path=rtorrent-ps_0.9.6-PS-1.1-25-g1b53ac9%7Ebionic_amd64.deb

**Comment by Pingblue** [comment]
Thanks, but ended up just using 0.9.7 and fixing the connection issue by changing from port to Unix socket.

**Comment by milesbenson** [comment]
Is there a way to compile 0.9.6 on Ubuntu 18.04? Same error here.

**Comment by jtvd78** [comment]
I had this same problem. Fixed with building 0.9.8
